golem-worker-executor-base/src/durable_host/golem/v1x.rs

```diff
@@ -61,72 +64,72 @@ impl<Ctx: WorkerCtx> HostGetWorkers for DurableWorkerCtx<Ctx> {
         &mut self,
         self_: Resource<GetWorkers>,
     ) -> anyhow::Result<Option<Vec<WorkerMetadata>>> {
-        golem::api0_2_0::host::HostGetWorkers::get_next(self, self_)
+        golem_api_0_2_x::host::HostGetWorkers::get_next(self, self_)
             .await
             .map(|x| x.map(|x| x.into_iter().map(|x| x.into()).collect()))
     }
 
     async fn drop(&mut self, rep: Resource<GetWorkers>) -> anyhow::Result<()> {
-        golem::api0_2_0::host::HostGetWorkers::drop(self, rep).await
+        golem_api_0_2_x::host::HostGetWorkers::drop(self, rep).await
     }
 }
 
 #[async_trait]
 impl<Ctx: WorkerCtx> Host for DurableWorkerCtx<Ctx> {
     async fn create_promise(&mut self) -> anyhow::Result<PromiseId> {
-        golem::api0_2_0::host::Host::create_promise(self)
+        golem_api_0_2_x::host::Host::create_promise(self)
             .await
             .map(|x| x.into())
     }
 
     async fn await_promise(&mut self, promise_id: PromiseId) -> anyhow::Result<Vec<u8>> {
-        golem::api0_2_0::host::Host::await_promise(self, promise_id.into()).await
+        golem_api_0_2_x::host::Host::await_promise(self, promise_id.into()).await
     }
 
     async fn complete_promise(
         &mut self,
         promise_id: PromiseId,
         data: Vec<u8>,
     ) -> anyhow::Result<bool> {
-        golem::api0_2_0::host::Host::complete_promise(self, promise_id.into(), data).await
+        golem_api_0_2_x::host::Host::complete_promise(self, promise_id.into(), data).await
     }
 
     async fn delete_promise(&mut self, promise_id: PromiseId) -> anyhow::Result<()> {
-        golem::api0_2_0::host::Host::delete_promise(self, promise_id.into()).await
+        golem_api_0_2_x::host::Host::delete_promise(self, promise_id.into()).await
     }
 
     async fn get_oplog_index(&mut self) -> anyhow::Result<OplogIndex> {
-        golem::api0_2_0::host::Host::get_oplog_index(self).await
+        golem_api_0_2_x::host::Host::get_oplog_index(self).await
     }
 
     async fn set_oplog_index(&mut self, oplog_idx: OplogIndex) -> anyhow::Result<()> {
-        golem::api0_2_0::host::Host::set_oplog_index(self, oplog_idx).await
+        golem_api_0_2_x::host::Host::set_oplog_index(self, oplog_idx).await
```

Comment:
What was the need of the x? For the runtime to pick any 0.2.x version? 
We already changed it to 1.x right? So why is this part changing?